terraform_docs.sh

```diff
@@ -76,28 +72,25 @@ terraform_docs_() {
     local tmp_file_awk
     tmp_file_awk=$(mktemp "${TMPDIR:-/tmp}/terraform-docs-XXXXXXXXXX")
     terraform_docs_awk "$tmp_file_awk"
-    terraform_docs "$tmp_file_awk" "$args" "${files[@]}"
+    terraform_docs "$tmp_file_awk"
     rm -f "$tmp_file_awk"
 
   else # Using terraform 0.11 and no awk script is needed for that
 
-    terraform_docs "0" "$args" "${files[@]}"
+    terraform_docs "0"
 
   fi
 }
 
 terraform_docs() {
-  local -r terraform_docs_awk_file="$1"
-  local -r args="$2"
-  shift 2
-  local -a -r files=("$@")
+  local -r terraform_docs_awk_file="$1" ; shift
```

Comment:
There's not really much point in `shift` anymore, now that we are using the globals.
```suggestion
  local -r terraform_docs_awk_file="$1"
```

Reply:
> There's not really much point in `shift` anymore, now that we are using the globals.

This is more of a safety net - it will throw an error if there is no arg.